ui/admin/app/routes/onboarding/quick-setup/create-resources.js

```diff
@@ -79,8 +82,8 @@ export default class OnboardingQuickSetupCreateResourcesRoute extends Route {
     hostSet.host_catalog_id = hostCatalog.id;
     await hostSet.save();
     host.host_catalog_id = hostCatalog.id;
-    if (hostAddress) {
-      host.address = hostAddress;
+    if (targetAddress) {
+      host.address = targetAddress;
```

Comment:
I think we're trying to actually save the address to the target rather than using a host. The goal is to simplify the initial setup so we shouldn't be creating host catalogs, host sets, hosts, etc to add to the target

Reply:
Yes, missed that part and working on it ATM  😄 I will provide changes